django_project/base/models/project.py

```diff
@@ -104,8 +104,8 @@ class Project(models.Model):
 
     image_file = models.ImageField(
         help_text=_('A logo image for this project. '
-                    'Most browsers support dragging the image directly on to '
-                    'the "Choose File" button above.'),
+                    'The ideal size for your image '
```

Comment:
Why not keep both text hints (for drag and drop and size hint)

should be -> is